src/main/resources/languages/portugues-pt.yml

```diff
@@ -0,0 +1,212 @@
+BossBar:
+  DayTitle: "&eDia {DAY}"
+  NightTitle: "&9Noite {DAY}"
+Scoreboard:
+  DisplayName: "&6&lPrison Escape"
+  LastLine: "&eServidor de Teste"
+  GuardSideBar:
+    BalanceLine: "&fSaldo: &a{BALANCE}"
+    SoundDectorLine: "&fDetectores de Som:   "
+Items:
+  SelectPrisionerTeam:
+    Name: "&6Selecionar equipa dos prisioneiros"
+  SelectPoliceTeam:
+    Name: "&9Selecionar equipa dos polícias"
+  SelectNoneTeam:
+    Name: "&8Remover preferência"
+  Bolts:
+    Name: "Parafusos"
+  DuctTape:
+    Name: "Fita Isoladora"
+  Matches:
+    Name: "Fósforos"
+  Oil:
+    Name: "Óleo"
+  Stick:
+    Name: "Graveto"
+  PlasticPlate:
+    Name: "Placa de Plástico"
+  MetalPlate:
+    Name: "Placa de Metal"
+  Copper:
+    Name: "Cobre"
+  CircuitBoard:
+    Name: "Circuito Impresso"
```

Comment:
Placa de circuito*

Reply:
I believe that "placa de circuito" it's not used, the correct being "Circuito Impresso" or "Placa de Circuito Impresso".Please let me know if you really want the translation to be "Placa de Circuito".